CRM/Financial/BAO/PaymentProcessor.php

```diff
@@ -120,6 +120,26 @@ public static function getCreditCards($paymentProcessorID = NULL) {
     return [];
   }
 
+  /**
+   * Get options for a given contribution field.
+   *
+   * @param string $fieldName
+   * @param string $context see CRM_Core_DAO::buildOptionsContext.
+   * @param array $props whatever is known about this dao object.
+   *
+   * @return array|bool
+   * @see CRM_Core_DAO::buildOptions
+   *
+   */
+  public static function buildOptions($fieldName, $context = NULL, $props = []) {
```

Comment:
This change is to allow me to create processors with different financial account types in the unit test - that was what I stalled on when I first tried to deal with this